base/special/exp.jl

```diff
@@ -250,7 +250,7 @@ end
             twopk = (k + UInt64(53)) << 52
             return reinterpret(T, twopk + reinterpret(UInt64, small_part))*0x1p-53
         end
-        #k == 1024 && return (small_part * 2.0) * 2.0^1023
+        k == 1024 && return (small_part * 2.0) * 2.0^1023
```

Comment:
what's the case where this is necessary? I looked a bunch for it, but never could find it.

Reply:
`prevfloat(1.0) ^ ( log2(floatmax()) / log2(prevfloat(1.0))` delivered `NaN` instead of the expected `Inf`.